### PR TITLE
LPS-141939 | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-141939

cc @david-gutierrez-mesa 
cc @Tim-Cao

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/WidgetPagesUI.testcase [hunk 1]
```diff
@@ -32,89 +32,115 @@ definition {
...
 	@priority = "3"
 	test ViewOrderOfContentWidgetActions {
-		task ("Add a content page") {
+		task ("Add a widget page") {
 			JSONLayout.addPublicLayout(
 				groupName = "Test Site Name",
-				layoutName = "Test Widget Page Name");
+				layoutName = "Test Page Name");
+
+			Refresh();
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/WidgetPagesUI.testcase:139]: SitePages.openPagesAdmin(siteURLKey = "test-site-name");

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/WidgetPagesUI.testcase [hunk 2]
```diff
@@ -32,89 +32,115 @@ definition {
...
+			Page.view(
+				pageName = "Test Private Page",
+				privacy = "Private Pages");
+		}
 	}
 
+	@description = "Can add and delete a web content display widget via page."
 	@priority = "5"
+	@uitest
 	test AddThenDeleteWidgetFromPage {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/WidgetPagesUI.testcase:100]: AddAndDeleteWidget

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/WidgetPagesUI.testcase [hunk 3]
```diff
@@ -32,89 +32,115 @@ definition {
...
+		task ("Add three widget pages") {
+			for (var i : list "1,2,3") {
+				SitePages.addPublicPage(pageName = "Test Page Name ${i}");
+			}
+		}
 
-		Page.view(
-			pageName = "Site Page",
-			privacy = "Private Pages");
+		task ("View public pages in pages administration") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/WidgetPagesUI.testcase:73]: View pages in Pages admin

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/WidgetPagesUI.testcase [hunk 4]
```diff
@@ -32,89 +32,115 @@ definition {
...
+		}
+
+		task ("View Web Content Display is not shown") {
+			Portlet.viewNoSpecificPG(portletName = "Web Content Display");
+		}
 	}
 
 	@description = "This is a test for LPS-132422. The list of actions appear in right order for pages."
 	@priority = "3"
 	test ViewOrderOfContentWidgetActions {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/WidgetPagesUI.testcase:133]: ViewOrderOfPageActions

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/WidgetPagesUI.testcase [hunk 5]
```diff
@@ -32,89 +32,115 @@ definition {
...
+	test AddPrivatePageInNewSite {
+		task ("Add a private page") {
+			SitePages.addPrivatePage(pageName = "Test Private Page");
+		}
 
-		Page.view(
-			depth = "2",
-			pageName = "Site Child Page",
-			privacy = "Public Pages");
+		task ("View private page in pages administration") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/widgetpage/WidgetPagesUI.testcase:90]: View private page in Pages admin

**Comment by ozysouza** [comment]
> Hi @ozysouza, Could you update according to comments? Thanks!

Hi @Tim-Cao, sure! PR updated, could you take a look at? Thanks!

**Comment by Tim-Cao** [comment]
Hi @ozysouza, see changes https://github.com/liferay-echo/liferay-portal/compare/6cde6aeda3a84d961033f22973c02dc8bc47b66f...fd181a2a624aec5d9e09a068e1d570cca14830ae

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**